3600/3/helpers/comment_helper.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module CommentsHelper
+  def add_new_comment
+    @user = User.find(session[:user_id])
+    @comment = @user.comments.create(grade: params[:grade].to_i, text: params[:text])
```

Comment:
```ruby
@user.comments.create(grade: params[:grade].to_i, text: params[:text], restaurant_id: @restaurant.id)
```
Looks better, don't it?

Reply:
This line will be longer than 100 characters.